src/actions/start/start.action.ts

```diff
@@ -1,295 +1,27 @@
-import Enquirer from 'enquirer';
-import { isValidVariableName } from '../../utils/globalValidators.helper';
-import {
-  printWithBadge,
-  printWithMantisGradient,
-} from '../../utils/prettyPrint.helper';
+import { execa } from 'execa';
 import { Action } from '../abstract.action';
-import { StartCommandOptions } from './start.types';
-import {
-  changeDirectory,
-  createDirectory,
-  moveFile,
-  removeFile,
-  replaceInFile,
-  replaceInFiles,
-} from '../../utils/files.helper';
-import {
-  checkPortAvailability,
-  execCommand,
-  findAvailablePort,
-} from '../../utils/process.helper';
-import fs from 'fs';
-import path from 'path';
 
 export default class StartAction extends Action {
-  private options: StartCommandOptions;
-  private createMobileApp: boolean = false;
-
-  constructor(options: StartCommandOptions) {
+  constructor() {
     super('[START-ACTION]');
-    this.options = options;
   }
 
   async execute() {
-    printWithMantisGradient(`🛖   NX WORKSPACE CREATION   🛠️`);
-
-    this.logger.debug(`Started [START] Action...`, this.options);
-
-    console.log(printWithBadge({ text: 'Hello World !' }));
-
-    const { workspace, createMobileApp } =
-      this.options && this.options.workspace
-        ? this.options
-        : await this.getWorkspaceInfo();
-
-    this.createMobileApp = createMobileApp || false;
-
-    this.logger.debug(`Prompts Answers: ${workspace} - ${createMobileApp}`);
-
+    this.logger.debug(`Started [START] Action...`);
     try {
-      this.createNxWorkspace(workspace);
-      changeDirectory(path.resolve(workspace));
-      this.installDependencies();
-      this.createApplications(workspace);
-      this.createLibraryDirectories();
-      this.installAngularPackage();
-      this.generateFeatureLibrary();
-      this.moveContent(workspace);
-      this.cleanupDirectories(workspace);
-      this.replaceHomePage();
-      this.updateImportPaths(workspace);
-      // this.installConcurrentlyPackage();
-      // await this.checkPorts();
-      this.launchApplications();
-
-      this.logger.info('Workspace setup complete!');
-    } catch (error) {
-      this.logger.error(
-        `Error occurred during workspace setup: ${error.message}`,
+      await execa(
+        'npx',
+        ['nx', 'run-many', '--target=serve', '--projects=app-web,app-mobile'],
+        {
```

Comment:
The command is a little more complex than this. Could we support running the local db as well if selected?

```js
    await execa(
      'npx',
      [
        'nx',
        'run-many',
        startLocalDb ? '--targets=serve,start-local-db' : '--target=serve',
        '--projects=web-client,mobile-client',
      ],
      { cwd: workspacePath, stdio: 'inherit' },
    );
```